### DISCUSSION TITLE
Carousel - Is there a way to show the other items by default?

### DISCUSSION THREAD

**Discussion by wpfblendperson** [AvaloniaUI/Avalonia#16130] [General]
I want to make a control like a roulette, and the closest one is a carousel except I need to be able to show the items before and after the selected item and the items spin up and down when I click above or below the selected item... How can I show a limited number of items before and after the selected item? 
![image](https://github.com/AvaloniaUI/Avalonia/assets/45245867/03d880ab-6996-40ae-b922-b60c969bca5c)

**Comment by thevortexcloud** [comment]
Can't you just reuse a listbox for this with a custom theme?

**Reply by wpfblendperson** [reply]
it has more work to do like transitioning and rotating... the carousel has transition/rotate by default i just need to show the before and after items...

**Comment by kyuranger** [comment]
I am not sure whether it can solve your problem but you can set the SelectedIndex to it.
